constants.xml

```diff
@@ -288,7 +288,7 @@
     </recovery>
     <smart_contract>
         <ENABLE_SC>true</ENABLE_SC>
-        <SCILLA_ROOT>/scilla</SCILLA_ROOT>
+        <SCILLA_ROOT>/home/stephen/run/scilla</SCILLA_ROOT>
```

Comment:
and here too

Reply:
This should not be committed, good catch, no changes to constants.xml required. We are looking at changing the way we use it locally so you don't need to keep editing.